<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oublie</class>
 <widget class="QDialog" name="oublie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1906</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#page{

}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>-40</y>
     <width>1961</width>
     <height>1061</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#page_3{
	background-image: url(:/img/image/changer.png);
}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="styleSheet">
     <string notr="true">#page{
	background-image: url(:/img/image/chercher.png);
}</string>
    </property>
    <widget class="QPushButton" name="rechercher">
     <property name="geometry">
      <rect>
       <x>712</x>
       <y>650</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_oub">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>471</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: transparent;
    border: none; /* Optionnel : pour retirer la bordure */
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="styleSheet">
     <string notr="true">#page_2{
	background-image: url(:/img/image/repondez.png);
}</string>
    </property>
    <widget class="QPushButton" name="continuer">
     <property name="geometry">
      <rect>
       <x>722</x>
       <y>680</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Réinitialiser votre mot de passe</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_question">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>450</y>
       <width>371</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gadugi</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>question</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="reponse">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>561</y>
       <width>461</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: transparent;
    border: none; /* Optionnel : pour retirer la bordure */
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Réinitialiser votre mot de passe</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pass">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>451</y>
       <width>401</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: transparent;
    border: none; /* Optionnel : pour retirer la bordure */
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pass_2">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>561</y>
       <width>401</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: transparent;
    border: none; /* Optionnel : pour retirer la bordure */
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enregistrer">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>717</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="annuler">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>680</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
